server/protocols/webadmin/webadmin-mailbox/pom.xml

```diff
@@ -68,6 +68,10 @@
             <scope>test</scope>
             <type>test-jar</type>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-mailbox-store</artifactId>
```

Comment:
Looks very suspicious to me.

Reply:
@Arsnael plans to do a `mailbox-tools-quota` maven package that would replace this awefull mailbox-store dependency.

This is a proposal that I personally support.

Reply:
Done